src/Propel/Runtime/Collection/CollectionIterator.php

```diff
@@ -229,20 +229,26 @@ public function append($value)
     }
 
     /**
+     * @param int $sort_flags
+     *
      * @return void
      */
-    public function asort()
+    public function asort($sort_flags = SORT_REGULAR)
```

Comment:
in this case we probably can't get rid of this.
because this function overrides native `\ArrayIterator` and in PHP8 there is strict signature compatibility needed.